<launch>
    <node name="runDobot" pkg="dobot" type="runDobot" output="screen">
        <rosparam file="$(find dobot)/params/zero.yaml" command="load" />
        <rosparam file="$(find dobot)/params/limit.yaml" command="load" />
    </node>
</launch>
